step3/index.js

```diff
@@ -0,0 +1,103 @@
+const express = require('express')
+const app = express()
+const port = 3000
+
+const mongoose = require('mongoose')
+const config = require('./config/key') // mongoDB URI를 보호하기 위한 설정파일
+const { Item } = require('./Item')
+
+app.use(express.json()) // body-parser 대신 사용
+
+mongoose.connect(config.mongoURI, {
+    useNewUrlParser: true, useUnifiedTopology: true, useCreateIndex: true, useFindAndModify: false
+}).then(() => console.log('MongoDB Connected...'))
+    .catch(err => console.log(err))

```

Comment:
```suggestion
mongoose.connect(config.mongoURI, {
  useNewUrlParser: true,
  useUnifiedTopology: true,
  useCreateIndex: true,
  useFindAndModify: false
})
  .then(() => console.log('MongoDB Connected...'))
  .catch(err => console.log(err))
```
개행을 적당히 잘 해주면 좋을 것 같아요!

Reply:
```js
  .catch(err => console.error(err))
```
보통 error의 경우 console.log 대신에 console.error를 사용한답니다!